tools/parallelize/parallelize.cpp

```diff
@@ -148,11 +149,29 @@ void schedule_command(parallelizer& p, const std::wstring_view commandPrefix, co
 
 extern "C" int wmain(int argc, wchar_t* argv[]) {
     try {
+        using namespace std::string_view_literals;
+
+        static constexpr std::array accepted_extensions{
+            L""sv, // extensionless headers like <vector>
+            L".cpp"sv,
+            L".h"sv,
+            L".hpp"sv,
+        };
+
+        // TRANSITION, P0202R3, use constexpr is_sorted()
+        assert(std::is_sorted(accepted_extensions.begin(), accepted_extensions.end()));
+
         if (argc < 3) {
             puts("Usage: parallelize.exe commandPrefix pathRoot0 [... pathRootN]\n"
                  "The command:\n"
                  "commandPrefix file\n"
-                 "will be launched in parallel for every regular file under any of pathRoots, recursively.");
+                 "will be launched in parallel for regular files under any of pathRoots, recursively.");
+            printf("Accepted extensions: ");
+            for (const auto& sv : accepted_extensions) {
+                printf("\"%.*ls\", ", static_cast<int>(sv.size()), sv.data());
+            }
+            printf("\n");

```

Comment:
Why not put this \n in the following puts?

Reply:
Stylistic preference: I don’t consider it to be “attached” to that line.

Reply:
(Originally, I printed both accepted and rejected extensions, so being able to copy the code blocks was nicely symmetric. I can fuse this if you think it would be better.)

Reply:
I admit that I also found the `puts; printf; /* loop */ printf; puts;` structure odd, but didn't bother to comment since I don't think this program will be limited by output speed.

Reply:
I was tempted to convert them all to `printf` for consistency, but refrained from doing so.